controller/array_action/array_mediator_svc.py

```diff
@@ -1,7 +1,51 @@
 from controller.array_action.array_mediator_interface import  ArrayMediator
+from controller.common.csi_logger import get_stdout_logger
+from controller.array_action.errors import StorageClassCapabilityNotSupported,PoolDoesNotMatchCapabilities
+
+array_connections_dict = {}
+logger = get_stdout_logger()
 
 class SVCArrayMediator(ArrayMediator):
     max_connections = 10  # TODO : need to implement all the interface methods\properties 
     port = 22
     array_type = "svc"
     pass
+
+    def validate_supported_capabilities(self, capabilities, pool=None):
+        """
+        :param capabilities: supportted capabilities is one of [{"SpaceEfficiency": "Thin"},\
+                                                                {"SpaceEfficiency": "Thick"},\
+                                                                {"SpaceEfficiency": "Compression"},\
+                                                                {"SpaceEfficiency": "Dedup"}]
+        :param pool: pool name
+        :return:
+        """
+        logger.info("validate_supported_capabilities for capabilities : {0}".format(capabilities))
+        if (len(capabilities) > 0 and not capabilities.get('SpaceEfficiency')) or \
```

Comment:
replace "not capabilities.get('SpaceEfficiency')" with 'SpaceEfficiency' in capabilities